sources/packages/forms/src/form-definitions/institutionlocation.json

```diff
@@ -112,7 +112,27 @@
                 "when": null,
                 "eq": ""
             },
-            "logic": [],
+            "logic": [
+                {
+                    "name": "Client not AEST",
```

Comment:
The logic name can be more close to it's purpose. like "Disable inputs for non AEST Client".

Reply:
The logic name is Client not AEST, nut the action is Disable inputs

![image](https://user-images.githubusercontent.com/62901416/168397281-e19318ba-32c9-4a38-a69d-351eed52c16b.png)

Reply:
Actually the logic is trigger + action.

That is the name for whole logic, not trigger or action independantly.